<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LENOVO" failures="1" tests="1" name="firefoxTest" time="85.114" errors="0" timestamp="2024-02-21T01:51:19 IST">
  <testcase classname="smokeTests.taskScenario" name="placeOrderFunctionality" time="68.122">
    <failure type="java.lang.AssertionError" message="expected [We&amp;apos;ll email you an order confirmation with details and tracking info.4] but found [We&amp;apos;ll email you an order confirmation with details and tracking info.]">
      <![CDATA[java.lang.AssertionError: expected [We'll email you an order confirmation with details and tracking info.4] but found [We'll email you an order confirmation with details and tracking info.]
at pageObjects.Assertions.verifyOrderIsPlaced(Assertions.java:14)
at smokeTests.taskScenario.placeOrderFunctionality(taskScenario.java:71)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- placeOrderFunctionality -->
</testsuite> <!-- firefoxTest -->
